R/Rscape/build_tarballs.jl

```diff
@@ -0,0 +1,155 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, BinaryBuilderBase, Pkg
+
+name = "Rscape"
+version = v"2.0.0-k"
+
+file_version_str = if isempty(version.prerelease)
+    "$version"
+else
+    "$(version.major).$(version.minor).$(version.patch).$(first(version.prerelease))"
+end
```

Comment:
We can't really use prerelease tags in version numbers of packages, maybe one day we'll explicitly prohibit it.  I usually do the other way around: write down the version string used by upstream, and build up the version we want to use out of it, maybe something like
```julia
VersionNumber(replace(file_version_str, r"([0-9]+\.[0-9]+\.[0-9]+).*" => s"\1"))
```
?

Reply:
Thanks for the review!

I made the change, though i have to admit i think it would be nice to use the correct version numbers for binary packages. Not sure if that would be have to be done at the Pkg or the BinaryBuilder level. Or maybe there is something i am missing :)

Reply:
It's a limitation of Pkg, it doesn't really like anything outside of `X.Y.Z` versioning scheme, with JLL packages we're already outside of it with the use of the build number, which causes some issues.  Also, lots of projects have very wild versioning scheme, completely outside of the `X.Y.Z-prerelease+build` scheme, we can't possibly cater for all of them so we have to do some normalisation anyway.